lib/include/TShirts/Size.h

```diff
@@ -0,0 +1,32 @@
+// Copyright(c), Philips Medical Systems Nederland B.V., IGT-Devices
+
+#pragma once
+
+#include "SizeLimits.h"
+
+#include <cstdint>
+
+namespace TShirt {
+
+enum class Label
+{
+    Small,
+    Medium,
+    Large
+};
+
+class Size
```

Comment:
The `Size` class is definitely a good idea. Using a custom type instead of native type is a good approach 👍 
Question on this class: Does it really encapsulate the size? Suppose we need to add a size like `XL`. The change would span multiple classes, I guess?
Can you think of a way to "add" something to enable an "added" size, without having to "change" the code in one or more classes?